com.minres.coredsl/src/com/minres/coredsl/services/visualization/Visualizer.xtend

```diff
@@ -481,7 +496,7 @@ class Visualizer {
 	}
 	
 	private def dispatch VisualNode genNode(StringLiteral node) {
-		return makeImmediateLiteral(node.value)
+		return makeImmediateLiteral(node.value);
```

Comment:
Fine for now, but a point for #10 -- Consistent use of semicolons in Xtend code.